src/app/component/ProductWishlistButton/ProductWishlistButton.component.js

```diff
@@ -97,10 +97,10 @@ export class ProductWishlistButton extends PureComponent {
     }
 
     renderLoader() {
-        const { isLoading } = this.props;
+        const { isLoadingLocal } = this.props;
```

Comment:
Why `Local`?

Reply:
Because it is kept in the local state.

Reply:
What is the difference between `isLoading` and `isLoadingLocal`? If they reflect the loading process of different things - prefer using `is<Thing>Loading`

Reply:
isLoading is from global state, it changes to true when addtowishlist button is pressed and to false when item was added to wishlist (it takes some time). isLoadingLocal is for each addtowishlist button local state. I did it to fix this PR bug: "Now when the wishlist button is pressed, the loading icon is displayed on all buttons." (See on top)